common/src/main/java/com/alibaba/fescar/common/loader/EnhancedServiceLoader.java

```diff
@@ -100,6 +102,28 @@
         return loadFile(service, activateName, loader);
     }
 
+    /**
+     * 返回所有实现类实例
```

Comment:
comment this in English.

Reply:
@jovany-wang   I will change it to English, but there were many Chinese comments in this file before.